app.js

```diff
@@ -1,13 +1,14 @@
 import express from 'express';
 import bodyParser from 'body-parser';
-import router from './server/routes/defaultRoutes';
+import { orderRoutes, defaultRoutes } from './server/routes/index';
```

Comment:
I don't think you need to include **'./server/routes/index'** the index, **./server/routes/** this should be fine

Reply:
Thanks for the observation. I've implemented your suggestion and it works very fine.